docs/src/installation.md

```diff
@@ -0,0 +1,224 @@
+<!-- spell-checker:ignore UTILNAME DESTDIR -->
```

Comment:
this is a bunch of duplication, would it be possible to avoid that?

Reply:
I didn't think it was possible at first, but I found an mdbook feature to get the instructions directly from the README.md!